### PR TITLE
Update introduction page for v4

### LINKED ISSUE #909: V4 - Update "Introduction" page
We should update the "[Introduction](https://deploy-preview-819--docs-oceanprotocol.netlify.app/concepts/introduction/)" page as well. thinking especially at these paragraphs:
* "In Ocean Protocol, each data service gets its own [ERC20](https://github.com/ethereum/EIPs/blob/7f4f0377730f5fc266824084188cc17cf246932e/EIPS/eip-20.md) datatoken. This enables data wallets, data exchanges, and data co-ops by directly leveraging crypto wallets, exchanges, and more.".
* "Publish and consume data services: downloadable files or compute-to-data. Use Ocean to deploy a new ERC20 datatoken contract for each data service, then mint datatokens."
* "And more. Use ERC20 support in [web3.js](https://web3js.readthedocs.io/), [web3.py](https://web3py.readthedocs.io/en/stable/examples.html#working-with-an-erc20-token-contract) and Solidity to connect datatokens with crypto wallets and other DeFi services."
Now that we'll have ERC721

### PR DESCRIPTION
**Pull request by akshay-ap**
Fixes #909

Changes proposed in this PR:

- Update introduction page for v4

### CODE REVIEW

File: content/concepts/introduction.md [hunk 1]
```diff
@@ -3,20 +3,23 @@ title: Introduction
...
 
-**Manage Datatokens for use in DeFi.** Use Ocean [JavaScript](https://github.com/oceanprotocol/ocean.js) or [Python](https://github.com/oceanprotocol/ocean.py) drivers to manage datatokens:
+**Manage Datatokens and Data NFTs for use in DeFi.** Use Ocean [JavaScript](https://github.com/oceanprotocol/ocean.js) or [Python](https://github.com/oceanprotocol/ocean.py) drivers to manage data NFTs and datatokens:
 
-- _Publish and consume data services:_ downloadable files or compute-to-data. Use Ocean to deploy a new [ERC20](https://github.com/ethereum/EIPs/blob/7f4f0377730f5fc266824084188cc17cf246932e/EIPS/eip-20.md) datatoken contract for each data service, then mint datatokens.
+- _Publish and consume data services:_ downloadable files or compute-to-data. Use Ocean to deploy a new [ERC721](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md and [ERC20](https://github.com/ethereum/EIPs/blob/7f4f0377730f5fc266824084188cc17cf246932e/EIPS/eip-20.md) datatoken contract for each data service, then mint datatokens.
```
Comment (AnaLoznianu) [inline content/concepts/introduction.md:16]: Remove the space between [ERC721] and the link.
The link and the displayed text are not formatted properly 
Screenshot 2022-03-11 at 14 19 42

File: content/concepts/introduction.md [hunk 2]
```diff
@@ -3,20 +3,23 @@ title: Introduction
 description: Ocean Protocol - Tools for the Web3 Data Economy
 ---
 
-In Ocean Protocol, each data service gets its own ERC20 **datatoken**. This enables data wallets, data exchanges, and data co-ops by directly leveraging crypto wallets, exchanges, and more.
+In Ocean Protocol, each data service gets its own ERC721 **data NFT** and one or more ERC20 **datatoken**. This enables data wallets, data exchanges, and data co-ops by directly leveraging crypto wallets, exchanges, and more.
```
Comment (trentmc) [inline content/concepts/introduction.md:6]: "datatoken" -> "datatokens"

**Review by trentmc — CHANGES_REQUESTED** [review]

File: content/concepts/introduction.md (hunk 2 above)
Comment (akshay-ap) [reply content/concepts/introduction.md]: done

File: content/concepts/introduction.md (hunk 1 above)
Comment (akshay-ap) [reply content/concepts/introduction.md]: done

**Review by AnaLoznianu — APPROVED** [review]

**Pull request merged**